templates/user/participation.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends "user/user_layout.html.twig" %}
+
+{% block content %}
+    <div class="container-fluid row">
+        <div class="col-6">
+            {% if upcomingConferences|length %}
+                <div class="row mb-5">
+                    <h2 class="mx-0">Conferences you will attend soon</h2>
+                    {% for conference in upcomingConferences %}
+                        <div class="pl-5">
+                            <a href="{{ path('conferences_show', {slug: conference.slug}) }}">{{ conference.name }}</a>, starts at {{ conference.startAt|date('m/d/Y') }}
+                        </div>
+                    {% endfor %}
+                </div>
+            {% endif %}
+
+            {% if pastConferences|length %}
+                <div class="row mb-5">
+                    <h2 class="mx-0">Conferences you went to</h2>
+                    {% for conference in pastConferences %}
+                        <div class="pl-5">
+                            <a href="{{ path('conferences_show', {slug: conference.slug}) }}">{{ conference.name }}</a>, starts at {{ conference.startAt|date('m/d/Y') }}
+                        </div>
+                    {% endfor %}
+                </div>
+            {% endif %}
+        </div>
+        <div class="col-6">
+            {% include "user/form/participation_form.html.twig" %}
```

Comment:
include tag is deprecated, you must use the include function instead

Reply:
It is for now, but I'm planning on using it again in the near future

Reply:
Resolving and letting this untouched since it should be used soon